src/sentry/issues/grouptype.py

```diff
@@ -18,6 +28,22 @@ class GroupCategory(Enum):
 _group_type_registry: Dict[int, Type[GroupType]] = {}
 _slug_lookup: Dict[str, Type[GroupType]] = {}
 _category_lookup: Dict[int, Set[int]] = defaultdict(set)
+DEFAULT_IGNORE_LIMIT: int = 3
+DEFAULT_EXPIRY_TIME: int = 60 * 60 * 24
+
+
+@dataclass(frozen=True)
+class NoiseConfig:
+    ignore_limit: int = DEFAULT_IGNORE_LIMIT
+    expiry_time: int = DEFAULT_EXPIRY_TIME
+
+
+@dataclass(frozen=True)
+class GroupPolicy:
+    feature: Optional[str] = None  # should be in the format "organizations:name-of-feature"
```

Comment:
I might want to move `feature` up to the `GroupType` level in the next PR. would make more sense in an automated system instead of just in a `GroupPolicy` context

Reply:
we can...should be fine as long as we don't need to call out the flags in code